.golangci.yml

```diff
@@ -50,31 +63,31 @@ linters:
     - govet
     # - ifshort
     - ineffassign
-    # - importas
+    - importas
     # - lll
-    # - misspell
+    - misspell
     - makezero
-    # - nakedret
-    # - nilerr
+    - nakedret
+    - nilerr
     # - nilnil
     # - nlreturn
     - noctx
     - nolintlint
     # - paralleltest
     - predeclared
-    # - rowserrcheck
+    - rowserrcheck
     - staticcheck
     - stylecheck
-    # - sqlclosecheck
+    - sqlclosecheck
     # - tagliatelle
-    # - tenv
-    # - thelper
-    # - tparallel
+    - tenv
+    - thelper
+    - tparallel
     - typecheck
     - unconvert
-    # - unparam
-    - unused
-    # - wastedassign
+    - unparam
+    # - unused  # lots of unused structs and methods
```

Comment:
How so? Shall we just remove them?

Reply:
The entire checkpoint.go is unused.

Reply:
Just so we can have the linter checking other code, it would be good to enable it, but disable it specifically for checkpoint.go.

Reply:
Done.